helm/README.md

```diff
@@ -192,12 +191,12 @@ To use this feature, use a container image containing `torch` (PyTorch) instead
 Use the Voxel51 provided image `fiftyone-app-torch` or build your own base image including `torch`.
 
 To override the default image, add a new `appSettings.image.repository` stanza to the Helm Chart.
-Using the included `values.yaml` this configuration might look like:
+For example, `values.yaml` might look like:
 
 ```yaml
 appSettings:
   image:
-    repository: voxel51/fiftyone-app-torch
+    repository: voxel51/fiftyone-app-torch:v1.4.1
```

Comment:
you don't put a version number on the repository line of a `values.yaml` - that would go on the `tag:` line.  The Helm chart will, by default, deploy the version of the container that is associated with the version of the Helm chart.

```suggestion
    repository: voxel51/fiftyone-app-torch
```